types/bunyan/index.d.ts

```diff
@@ -270,6 +270,29 @@ declare namespace Logger {
         destroy(): void;
         destroySoon(): void;
     }
+
+    interface RotatingFileStreamOptions {
+        path?: string;
```

Comment:
I don't think `path` is [optional](https://github.com/trentm/node-bunyan/blob/fe31b83e42d9c7f784e83fdcc528a7c76e0dacae/lib/bunyan.js#L1237)

Reply:
@amikhalev I agree, already corrected.